common/changes/@microsoft/api-documenter/ianc-upgrade-node-typings_2020-01-15-10-32.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Upgrade Node typings to Node 10",
+      "type": "minor",
```

Comment:
Maybe this should be `"patch"` -- the API doesn't seem to depend on NodeJS at all